src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -205,6 +205,7 @@
                                 <div id="commentTextEdit<%=commentIdx%>" style="display:none;">
                                     <div class="form-group form-inline">
                                         <div class="form-group text-muted">
+                                            Comment about <%=InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name)%>:<br>
```

Comment:
u used consecutive <br> for instructorFeedbackResults*.jsp, but not for instructorStudentRecords.jsp.

btw it's not a good practice to use <br> here
> Do not use <br> to increase the gap between lines of text; use the CSS margin property or the <p> element.
fyi: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br